subsys/fw_info/fw_info.c

```diff
@@ -101,14 +105,19 @@ static const struct fw_info_ext_api_request *skip_ext_apis(
 }
 
 
+#ifndef CONFIG_EXT_API_PROVIDE_EXT_API_REQUIRED
```

Comment:
Could you explain why we have this logic? To me it is not clear why we have these two variants

Reply:
Inside this `#ifndef` is the proper implementation of `fw_info_ext_api_provide()`. If `CONFIG_EXT_API_PROVIDE_EXT_API_REQUIRED` is `y`, we want to call the EXT_API instead, which happens inside the `#else`.

I'll add this in a comment in the code as well :).

Reply:
Added

Reply:
Just for my own understanding, would mcuboot still be capable of exposing its own EXT_API?

Reply:
Not really as it is right now.

Reply:
> > > I don't understand this feature enough to approve, so a bit of education is needed :)
> > 
> > 
> > See description :).
> 
> Would it make sense to include that info in the commit message, or the code somehow?

Good point, added to commit message.

Reply:
Should I add the possibility for MCUboot to share its own APIs? Or should we do it later?

Reply:
Lets wait with that.